pkg/cmd/admin/policy/reconcile_clusterrolebindings.go

```diff
@@ -143,10 +144,13 @@ func (o *ReconcileClusterRoleBindingsOptions) Validate() error {
 	return nil
 }
 
-// ReconcileClusterRoleBindingsOptions contains all the necessary functionality for the OpenShift cli reconcile-cluster-role-bindings command
 func (o *ReconcileClusterRoleBindingsOptions) RunReconcileClusterRoleBindings(cmd *cobra.Command, f *clientcmd.Factory) error {
 	changedClusterRoleBindings, err := o.ChangedClusterRoleBindings()
-	if err != nil {
+	if IsClusterRoleBindingLookupError(err) {
```

Comment:
```go
if err != nil && !IsClusterRoleBindingLookupError(err){
    return err
}
```

Then instead of `return nil` below, return the error.  Choose a better name for it to indicate that its special.